### DISCUSSION TITLE
How do I upload a local repository from PC to GitHub

### DISCUSSION THREAD

**Discussion by Kaillr** [community/community#143510] [Programming Help] [answered]
### Body

How do I upload a local repository from PC to GitHub

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DenysDyshlovskyi** [comment, marked as answer]
use github desktop, and commit your changes, and then push to the main branch

**Reply by Kaillr** [reply]
Thank you 😊

**Reply by DenysDyshlovskyi** [reply]
> Thank you 😊

You're welcome
